src/modules/generalExtensions/main.ts

```diff
@@ -61,6 +61,16 @@ export default (async (LSSM, MODULE_ID, $m) => {
             )
         ).default(LSSM, mapUndo, ownMapMarkers, getSetting, MODULE_ID);
     }
+    if (
+        window.location.pathname === '/' &&
+        (await getSetting<boolean>('extensionCloseCall'))
+    ) {
+        await (
```

Comment:
The asset is not async so the await is optional. Just as a side note for you, don't worry about it 😉